test/exporter_test_base.py

```diff
@@ -35,3 +37,90 @@ def __init__(self):
         self.addon_path = args.addon_path
 
         debug_utils.setup_logging(os.path.join(self.working_dir, 'debug.txt'))
+
+    def get_exportable_scenes_for_test(self,
```

Comment:
Can't call this from inherited classes. "No attribute 'get_exportable_scenes_for_test'". Does not work with super().get_exportable_scenes_for_test either, did I miss something?